src/main/java/games/strategy/engine/lobby/client/login/LobbyLogin.java

```diff
@@ -158,7 +162,11 @@ public void notifyFailedLogin(final String message) {
               final Map<String, String> props = new HashMap<>();
               props.put(LobbyLoginValidator.REGISTER_NEW_USER_KEY, Boolean.TRUE.toString());
               props.put(LobbyLoginValidator.EMAIL_KEY, createAccount.getEmail());
+              // TODO: Don't send the md5-hashed password once the lobby is updated
```

Comment:
Why have this TODO, the lobby *will* be updated first before we update game clients. We can change this directly? The key is to make sure that old game clients still work with the updated lobby.